src/types.rs

```diff
@@ -98,12 +119,75 @@ pub struct Assignment {
     pub name: String,
 }
 
-#[derive(Responder)]
-#[response(status = 400)]
-pub struct BadRequest(pub &'static str);
+// Input type for /hosts/stats endpoint
+// Data schema in collection `holoports_status`
+#[derive(Serialize, Deserialize, Clone)]
+#[serde(crate = "rocket::serde")]
+#[serde(rename_all = "camelCase")]
+pub struct HostStats {
+    pub holo_network: Option<String>,
+    pub channel: Option<String>,
+    pub holoport_model: Option<String>,
+    pub ssh_status: bool,
+    pub zt_ip: String,
+    pub wan_ip: String,
+    pub holoport_id_base36: String,
+    pub timestamp: String,
+    pub email: String, // >> discuss adding email to `HostStats` schema to use as filter instead of pubkey/hostid
```

Comment:
We cannot do it because holoport is not aware of a registration email. It only knows email address provided for holoport admin, but we cannot assume both are gonna stay the same for ever